core/src/script/CGXP/plugins/WFSPermalink.js

```diff
@@ -227,6 +236,10 @@ cgxp.plugins.WFSPermalink = Ext.extend(gxp.plugins.Tool, {
 
         var layerName = state.layer;
         delete state.layer;
+        if (state.show) {
+            this.showFeatures = false;
```

Comment:
OK...

I agree with you for 

```javascript
this.showFeatures = state.show !== "false";
```
But after that, we really have to add:
```javascript
if (state.show) {
    delete state.show;  
}
```
Because on <a href=https://github.com/camptocamp/cgxp/blob/master/core/src/script/CGXP/plugins/WFSPermalink.js#L240 target="_blank">line 240</a> we call `createFilter` to which we pass `state`.

Then in this function, on <a href=https://github.com/camptocamp/cgxp/blob/master/core/src/script/CGXP/plugins/WFSPermalink.js#L272 target="_blank">line 242</a>, we add all the content of `state` to a variable `params`.

A few lines down under, this one is then added to a variable `filter` and so on until the whole thing returns an `OpenLayers.Filter.Logical` object.

Then on <a href=https://github.com/camptocamp/cgxp/blob/master/core/src/script/CGXP/plugins/WFSPermalink.js#L241 target="_blank">line 241</a> we use this filter to make the WFS request.

If I do not do `delete state.show`, then my filter looks like this:
![bitmap](https://cloud.githubusercontent.com/assets/1681332/6730164/182a06aa-ce3a-11e4-8e56-3dd0b5fbe547.png)

And surely does my MapServer layer not contain such an attribute, thus doesn't the response contain anything....

So... I would suggest something like:
```javascript
        this.showFeatures = state.show !== "false";
        if (state.show) {
            delete state.show;  
        }
```